iceberg-catalog-migrator/cli/build.gradle.kts

```diff
@@ -150,7 +158,11 @@ val processResources =
 
 val mainClassName = "org.apache.polaris.iceberg.catalog.migrator.cli.CatalogMigrationCLI"
 
-val shadowJar = tasks.named<ShadowJar>("shadowJar") { isZip64 = true }
+val shadowJar =
+  tasks.named<ShadowJar>("shadowJar") {
+    isZip64 = true
+    from(project.projectDir) { include("LICENSE", "NOTICE") }
```

Comment:
Haven't checked the entries of the final jar, but I think we have to add [this snippet](https://github.com/apache/polaris/blob/4a80c513d142c6f9211bfc1e431794a7dd3ac1e9/plugins/spark/v3.5/spark/build.gradle.kts#L92-L98) as well.

Reply:
thx